api/src/models/issue.ts

```diff
@@ -0,0 +1,17 @@
+import mongoose, { Document, Schema } from 'mongoose';
+import { IIssue } from 'ssw-common';
+import { issueFormatEnum } from '../utils/enums';
+
+type IssueSchema = IIssue & Document<any>;
+/**
+ * Mongoose Schema to represent an Issue at South Side Weekly.
+ */
+const Issue = new mongoose.Schema({
+  name: { type: String, default: null, required: true },
+  deadlineDate: { type: String, default: null, required: true },
+  releaseDate: { type: String, default: null, required: true },
+  pitches: [{ type: Schema.Types.ObjectId, ref: 'Pitch' }],
+  type: { type: String, enum: Object.values(issueFormatEnum), required: true },
```

Comment:
If the enum is called `issueFormat` should one of the two (this property called `type` or `issueFormatEnum`) be renamed to match the other?